plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -810,6 +828,7 @@ private static void setDeltaStorageFormat(Table.Builder tableBuilder, String loc
     public DeltaLakeOutputTableHandle beginCreateTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, Optional<ConnectorTableLayout> layout, RetryMode retryMode)
     {
         validateTableColumns(tableMetadata);
+        checkChangeDataFeedColumnNames(tableMetadata.getColumns().stream().map(ColumnMetadata::getName).collect(toImmutableSet()));
```

Comment:
Why `checkChangeDataFeedColumnNames` isn't  part of `validateTableColumns`?
maybe we can inline it there?